src/test_library.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import json
+import os
+from unittest.mock import patch
+from main import (
```

Comment:
Nem zavart, hogy nem futnak le a tesztek vscode-ból?
```suggestion
from src.main import (
```